modules/apps_api/spec/requests/v0/application_directory_request_spec.rb

```diff
@@ -92,13 +105,19 @@
   end
 
   describe '#create /services/apps/v0/directory' do
+    before do
+      allow(Settings.directory).to receive(:secret).and_return(auth_string)
+    end
+
     it 'creates the app' do
+      allow(Settings.directory).to receive(:secret).and_return('blah')
```

Comment:
and line 120 should be covered in `before`-block.  If the before block is in all the `describe`-blocks, if can just be at the top-level one

Reply:
resolved in latest commit